test/spec/sizeMapping_spec.js

```diff
@@ -1,123 +1,105 @@
 import { expect } from 'chai';
-import * as sizeMapping from 'src/sizeMapping';
-
-var validAdUnit = {
-  'sizes': [300, 250],
-  'sizeMapping': [
-    {
-      'minWidth': 1024,
-      'sizes': [[300, 250], [728, 90]]
-    },
-    {
-      'minWidth': 480,
-      'sizes': [120, 60]
-    },
-    {
-      'minWidth': 0,
-      'sizes': [20, 20]
-    }
-  ]
-};
-
-var invalidAdUnit = {
-  'sizes': [300, 250],
-  'sizeMapping': {} // wrong type
-};
-
-var invalidAdUnit2 = {
-  'sizes': [300, 250],
-  'sizeMapping': [{
-    foo: 'bar' // bad
-  }]
-};
-
-let mockWindow = {};
-
-function resetMockWindow() {
-  mockWindow = {
-    document: {
-      body: {
-        clientWidth: 1024
-      },
-      documentElement: {
-        clientWidth: 1024
-      }
-    },
-    innerWidth: 1024
-  };
-}
-
-describe('sizeMapping', function() {
-  beforeEach(resetMockWindow);
-
-  it('minWidth should be inclusive', function() {
-    mockWindow.innerWidth = 1024;
-    sizeMapping.setWindow(mockWindow);
-    let sizes = sizeMapping.mapSizes(validAdUnit);
-    expect(sizes).to.deep.equal([[300, 250], [728, 90]]);
-  });
+import { resolveSizesFromLabels, setLabels, setSizeConfig } from 'src/sizeMapping';
+
+describe('sizeMapping', () => {
```

Comment:
should also test when multiple labels are passed into `resolveSizesFromLabels([multiple, labels], sizes);`